static/templates/index.html

```diff
@@ -10,6 +10,13 @@
       href="{{root}}/skin/index.css?KIWIXCACHEID"
       rel="Stylesheet"
     />
+    <link 
+      rel="alternate" 
+      type="application/rss+xml" 
+      title="Kiwix RSS Feed"
+      id="rssExtension" 
+      href="/catalog/search?"
```

Comment:
We better use the [new OPDS API](https://kiwix-tools.readthedocs.io/en/latest/kiwix-serve.html#catalog-v2-opds-api) (in this case - [`/catalog/v2/entries`](https://kiwix-tools.readthedocs.io/en/latest/kiwix-serve.html#catalog-v2-entries)).